tool/site/client-react/src/components/radioField.tsx

```diff
@@ -0,0 +1,31 @@
+import React, {FC} from "react"
+import {Radios} from "nhsuk-react-components"
+import {Field} from "formik"
+
+export interface RadioFieldProps {
+  name: string
+  label: string
+  fieldRadios: Array<Radio>
+}
+
+interface Radio {
+  value: string
+  text: string
+}
+
+const RadioField: FC<RadioFieldProps> = ({name, label, fieldRadios}) => (
+  <Field id={name} name={name} as={Radios}>
```

Comment:
This will give you a `<Radios>` tag inside a `<Radios>` tag. You just want the `<Field as={Radios}>` one. Pass it the other props you want